CMakeLists.txt

```diff
@@ -33,6 +37,19 @@ set(CPACK_NSIS_DISPLAY_NAME ${APPLICATION_NAME})
 set(CPACK_NSIS_INSTALLED_ICON_NAME ${EXECUTABLE_NAME}.exe)
 set(CPACK_NSIS_EXECUTABLES_DIRECTORY .)
 set(CPACK_NSIS_MUI_FINISHPAGE_RUN ${EXECUTABLE_NAME}.exe)
+
+# Setup installer exe properties
+set (CPACK_NSIS_EXTRA_DEFINES 
+   VIProductVersion \"${VERSION}\"
+   VIFileVersion    \"${VERSION}.0\"
+   VIAddVersionKey  \"ProductName\" \"${APPLICATION_NAME}\"
+   VIAddVersionKey  \"ProductVersion\" \"${VERSION}\"
+   VIAddVersionKey  \"CompanyName\" \"${COMPANY_NAME}\"
+   VIAddVersionKey  \"LegalCopyright\" \"${LEGAL_COPYRIGHT}\"
+   VIAddVersionKey  \"FileDescription\" \"${APPLICATION_NAME} Installer\"
+   VIAddVersionKey  \"FileVersion\" \"${VERSION}\"
```

Comment:
```suggestion
   VIAddVersionKey  \"FileVersion\" \"${CMAKE_PROJECT_VERSION}\"
```

Reply:
OK. Thanks.

Reply:
Pelase try to compile to check if it's OK.

Reply:
I currently don't have access to my Windows machine, this is why I asked you to try it yourself first.

Reply:
Ok. Please help me to fix the ECM error installing ECM manually.